geometry.zsh

```diff
@@ -217,10 +226,10 @@ prompt_geometry_render() {
   fi
 
   PROMPT="
- %${#PROMPT_SYMBOL}{%(?.$GEOMETRY_PROMPT.$GEOMETRY_EXIT_VALUE)%} %F{$GEOMETRY_COLOR_DIR}%3~%f "
+%${#PROMPT_SYMBOL}{%(?.$GEOMETRY_PROMPT.$GEOMETRY_EXIT_VALUE)%} %F{$GEOMETRY_COLOR_DIR}%3~%f "
 
   PROMPT2=" $GEOMETRY_SYMBOL_RPROMPT "
-  RPROMPT="$(prompt_geometry_git_info)%f"
+  RPROMPT="$(prompt_geometry_virtualenv) $(prompt_geometry_git_info)%f"
```

Comment:
Can you move the space between both calls to the `prompt_geometry_virtualenv` `echo` call? The way you put it, it will display that extra space even if users have virtualenv disabled.